docs/outputs.rst

```diff
@@ -22,207 +21,320 @@ FMRIPrep generates three broad classes of outcomes:
      the same-subject's T1w space or into MNI space.
 
   3. **Additional data for subsequent analysis**, for instance the transformations
-     between different spaces or the estimated confounds.
+     between different spaces or the estimated confounds_.
 
 
-fMRIPrep outputs conform to the :abbr:`BIDS (brain imaging data structure)`
+*fMRIPrep* outputs conform to the :abbr:`BIDS (brain imaging data structure)`
 Derivatives specification (see `BIDS Derivatives RC1`_).
 
 Visual Reports
```

Comment:
This section is almost too short, and doesn't provide that much information. Maybe a short overview/guide of how to review the reports would be useful?

Reply:
#1831